pyproject.toml

```diff
@@ -1,22 +1,45 @@
 [project]
 name = "dc_django_utils"
-version = "6.1.0"
+dynamic = ["version"]
 description = "This is a package to provide some common utilities for Django projects at Democracy Club."
 readme = "README.md"
 authors = [{name = "Sym Roe"}, {name = "Virginia Dooley"}]
 
 dependencies = [
-	"django>=3.2,<4.3",
-	"django-pipeline==3.0.0",
-	"markdown",
-	"django-localflavor==4.0",
-	"setuptools==69.2.0",
-	"whitenoise",
-	"pysass",
-	"jsmin<3.1",
+    "django>=3.2,<4.3",
+    "django-pipeline>=3.0.0",
+    "markdown",
+    "django-localflavor>=4.0",
+    "setuptools>=69.2.0",
+    "whitenoise",
+    "pysass",
+    "jsmin<3.1",
+    "pytidylib",
 ]
 
+[project.optional-dependencies]
+dev = [
+    "black==22.10.0",
+    "curlylint==0.13.1",
+    "isort==5.13.2",
+    "djhtml==3.0.6",
+    "ruff==0.6.1",
+    "pre-commit>=3.0.0",
+    "ipdb==0.13.13",
+    "pytest==8.3.2",
+    "pytest-django==4.8.0",
+    "pytest-mock==3.14.0",
+    "pytest-flakes==4.0.5",
+    "pytest-ruff==0.4.1",
+    "dc-design-system @ https://github.com/DemocracyClub/design-system/archive/refs/tags/0.5.0.tar.gz",
+]
+ci = [
+    "tox==4.18.0"
+]
 
+[tool.setuptools.dynamic]
+# set the package version number from __version__
+version = {attr = "dc_utils.__version__"}
```

Comment:
This implements the suggestion from https://github.com/DemocracyClub/dc_django_utils/issues/89
Closes https://github.com/DemocracyClub/dc_django_utils/issues/89